### ISSUE TITLE
No input signal detected

### ISSUE THREAD

**Issue by ppsgroup** [amiaopensource/vrecord#736]
Hi all,
We are running multiple seats of vrecord, and getting this error fairly often:
[decklink @ 0x5602d8a54a80] Frame received (#18452) - No input signal detected - Frames dropped 1
[decklink @ 0x5602d8a54a80] Frame received (#18454) - Input returned - Frames dropped 2

Here are the specs from 1 of the machines:
- Intel® Xeon(R) CPU W3565 
- 16GB RAM
OS: Ubuntu 20.04.2 LTS 64-bit
Vrecord Version: 2022-02-28
Blackmagic Driver: 12.4.1

Most of the Blackmagic cards are Decklink Mini Recorders, although there is at least 1 Decklink Studio 2 card.
The computers are HP Z400, and Z420 workstations.

Rebooting the workstation sometimes will fix the problem, but not every time.

We recently upgraded to the latest Blackmagic drivers, and Vrecord 2022-02-28, but are still having the same problem.
Any help would be greatly appreciated!

Best,
Jim Bird

**Comment by dericed** [comment]
Hey Jim, Could you test opening the Blackmagic Desktop Video Setup thing and setting the video input to match the one you're using in vrecord (like setting Desktop Video and vrecord to both be NTSC SD input, for example). Let us know if that makes an impact.

**Comment by ppsgroup** [comment]
Hi Dave,
We don't have an option to set that, only pick the input, SDI or HDMI.
Thanks,
Jim

![Screenshot from 2023-01-17 14-53-41](https://user-images.githubusercontent.com/109600752/213000194-a386e0f8-9ff6-498c-8c7b-61b1cd7c4a62.png)
![Screenshot from 2023-01-17 14-52-33](https://user-images.githubusercontent.com/109600752/213000200-c447260f-a5b7-4c11-bde0-898200b04c97.png)

**Comment by dericed** [comment]
Yes, I guess that's what I was remembering (the input rather than the format). Happy to see other linux users for vrecord. Any other ideas @privatezero ?

**Comment by ppsgroup** [comment]
There are no output options for the Decklink Mini, because it has no outputs.
We have that option on the Studio 2 card, and it is set to the same as your example, 525i59.94 NTSC.

Thanks,
Jim

**Comment by retokromer** [comment]
@privatezero : Should we update the [hardware](https://github.com/amiaopensource/vrecord/blob/main/Resources/Documentation/hardware.md) hints? (We don’t have any DeckLink Mini Recorder, therefore I cannot test this on my end.)
